src/server/stream_family.cc

```diff
@@ -1151,11 +1151,8 @@ struct PendingExtendedResult {
   mstime_t elapsed;
 };
 
-struct PendingResult {
-  bool is_reduced = true;
-  PendingReducedResult reduced;
-  vector<PendingExtendedResult> extended;
-};
+using PendingExtendedeResultList = std::vector<PendingExtendedResult>;
+using PendingResult = std::variant<PendingReducedResult, PendingExtendedeResultList>;
```

Comment:
You have an e here  PendingExtended**e**ResultList, seems like I added it accidentally when suggesting 😅